package.json

```diff
@@ -42,7 +42,7 @@
         "build": "yarn build:dev:dist",
         "build:ci:es": "yarn build:prod:es",
         "build:ci:dist": "LANGUAGE=en-US ENTRY=explorer REACT=true yarn build:prod:dist",
-        "build:dev:dist": "LANGUAGE=en-US BABEL_ENV=development NODE_ENV=development webpack --config scripts/webpack.config.js --mode development --progress --colors",
```

Comment:
is `--color` default on now?

Reply:
@greg-in-a-box ^

Reply:
it was giving an error saying it was deprecated as a object

Reply:
`--colors` or `--color`?

Reply:
https://github.com/rails/webpacker/issues/3138